core/templates/services/keyboard-shortcut.service.ts

```diff
@@ -124,10 +124,6 @@ export class KeyboardShortcutService {
       this.windowRef.nativeWindow.location.href = '/about';
     });
 
-    Mousetrap.bind('ctrl+5', () => {
-      this.windowRef.nativeWindow.location.href = '/notifications';
-    });
-
     Mousetrap.bind('ctrl+6', () => {
```

Comment:
Please also decrement these to `ctrl+5`

Reply:
Sure will do that, I was kinda confused to change it or not.